ingestion/src/metadata/data_insight/api/workflow.py

```diff
@@ -235,6 +237,17 @@ def _execute_kpi_runner(self):
                     "No sink attribute found, skipping ingestion of KPI result"
                 )
 
+    def _execute_web_analytics_event_data_cleaning(self):
+        """We will delete web analytics events older than `RETENTION_DAYS`
+        to limit its accumulation
+        """
+        tmps = get_beginning_of_day_timestamp_mill(days=RETENTION_DAYS)
+        for web_analytic_event in WebAnalyticEventType:
+            self.metadata.delete_web_analytic_event_before_ts_exclusive(
+                web_analytic_event,
+                tmps,
```

Comment:
nit:
```suggestion
        tmsp = get_beginning_of_day_timestamp_mill(days=RETENTION_DAYS)
        for web_analytic_event in WebAnalyticEventType:
            self.metadata.delete_web_analytic_event_before_ts_exclusive(
                web_analytic_event,
                tmsp,
```
to align it with the name we use on `ingestion/src/metadata/ingestion/ometa/mixins/data_insight_mixin.py`.